AnimationCodelab/finished/src/main/java/com/example/android/codelab/animation/ui/Color.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.android.codelab.animation.ui
+
+import androidx.compose.ui.graphics.Color
+
+val purple100 = Color(0xFFE1BEE7)
```

Comment:
These should all be pascal case.

Reply:
Then we might want to change the Android Studio new project template as well, as the template creates the colors with this casing!

Reply:
Filed https://issuetracker.google.com/issues/180697776

Reply:
Done